cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Operand.qll

```diff
@@ -25,7 +25,7 @@ private class TStageOperand =
  * (the defining instruction) in another instruction (the use instruction)
  */
 class Operand extends TStageOperand {
-  Operand() {
+  cached Operand() {
```

Comment:
I'm curious about whether caching at every level of the `Operand` hierarchy was necessary, or whether caching only the leaf classes would be sufficient. Performance numbers look great as-is, so no reason to change anything now. Just food for thought.